### PR TITLE
Fix stop words

### PR DESCRIPTION
**Pull request by ben-gready**

### CODE REVIEW

File: inst/SurveyAnalysisWebApp/app.R [hunk 1]
```diff
@@ -164,16 +165,29 @@ server <- function(input, output) {
...
                    label = "Set stop words for keywords",
                    choices = unique(tolower(selected_data()$watson_features$keywords$text)),
-                   selected = c("edmonton", "city", "great", "thanks", "things", "lack", "good work", "input", "right thing", "great place"),
+                   selected = c("edmonton", "city", "great", "thanks", "things", "lack", "good work", "input", "right thing", "great place", "lrt"),
                    multiple = TRUE, selectize = TRUE)
   })
+  
+  output$entities_drop <- renderUI({
+    selectInput(inputId = "stop_entities",
+                label = "Set stop words for ebtutues",
```
Comment (reisner) [inline inst/SurveyAnalysisWebApp/app.R]: @ben-gready hehe ebtutues

**Comment by ben-gready** [comment]
it was all in the way reactives were being passed to modules. Should now be working

File: R/Modules.R [hunk 2]
```diff
@@ -38,9 +38,13 @@ wordCloudSelector <-function(input, output, session, dt){
 
 #' @export
 wordCloudSelectorGetWord <- function(input,output,session){
+  word <- reactive(input$selected_word)
+  print(word)
```
Comment (reisner) [inline R/Modules.R]: @ben-gready do we still need the print statement?

File: R/Modules.R [hunk 3]
```diff
@@ -205,24 +211,19 @@ barChartView <-function(input, output, session, watson_dt, survey_dt){
 
   output$timeline_plot <- renderPlotly({
     s <- as.list(event_data("plotly_click"))
-    timeline <- watson_dt %>%
-      left_join(survey_dt %>%
-                  select(-response), by = "response_id") %>%
-      data.table
+    #print(s)
```
Comment (reisner) [inline R/Modules.R]: rm pls

File: inst/SurveyAnalysisWebApp/app.R [hunk 4]
```diff
@@ -128,70 +129,34 @@ server <- function(input, output) {
...
-  #                 "input", "right thing", "great place", "people", "time", "way", "opportunity",
-  #                 "example", "area", "ave", "issues","concern", "concerns", "block", "bit", "stop",
-  #                 "problems", "thing", "time", "end", "lets")
+  # defaults hard coded for now... eventually, should remember the stop words for each user/dataset and load them up.
+  # TODO: potentially, as Roman suggests, combine the keyword and entitiy stop words for simplicity... 
+  # would need to take the union of keywords & entities as the choices
+  stop_words <- c("edmonton", "city", "great", "thanks", "things", "lack", "good work",
+                  "input", "right thing", "great place", "people", "time", "way", "opportunity",
+                  "example", "area", "ave", "issues","concern", "concerns", "block", "bit", "stop",
+                  "problems", "thing", "time", "end", "lets")
```
Comment (reisner) [inline inst/SurveyAnalysisWebApp/app.R]: Actually the more I think about it, I think the entity and keywords should be separate stop lists. They are going to be different lists of words, and mean different things.

File: R/Modules.R [hunk 5]
```diff
@@ -255,7 +256,36 @@ chooseDataset <- function(input, output, session, store_loc, namespace_root){
...
+  ns <- NS(id)
+  tagList({uiOutput(outputId = ns("stop_words"))})
+}
+
+#' @export
+ApplyStopWords <- function(input, output, session, dt, initial_selection){
+  output$stop_words <- renderUI({
+    selectInput(inputId = session$ns("stop_words"),
+                label = "Set stop words for keywords",
+                choices = unique(tolower(dt$text)),
```
Comment (reisner) [inline R/Modules.R]: I get some duplicates that contain a period:

fullscreen_2018-01-05__8_44_am

File: R/Modules.R [hunk 6]
```diff
@@ -255,7 +256,36 @@ chooseDataset <- function(input, output, session, store_loc, namespace_root){
...
+ApplyStopWordsInput <- function(id){
+  ns <- NS(id)
+  tagList({uiOutput(outputId = ns("stop_words"))})
+}
+
+#' @export
+ApplyStopWords <- function(input, output, session, dt, initial_selection){
+  output$stop_words <- renderUI({
+    selectInput(inputId = session$ns("stop_words"),
+                label = "Set stop words for keywords",
```
Comment (reisner) [inline R/Modules.R]: This is called by both keywords and entities, so it shouldnt say "keywords".

File: R/Modules.R (hunk 5 above)
Comment (ben-gready) [reply R/Modules.R]: I will record this as a longer todo in trello...

File: inst/SurveyAnalysisWebApp/app.R (hunk 4 above)
Comment (ben-gready) [reply inst/SurveyAnalysisWebApp/app.R]: sounds good.. we can always change our minds later

**Comment by ben-gready** [comment]
as discussed, I am incorporating the above in new branch: https://github.com/ben-gready/free_text_analyzer/tree/new-data-format. I will put in a PR once complete

**Pull request merged**